.github/workflows/publish.yml

```diff
@@ -21,7 +21,19 @@ jobs:
         run: npm install
       - name: NPM run build
         run: npm run build
-      - name: Publish to NPM
+      - name: Get npm tag name if needed
```

Comment:
Could we remove the "if needed" in the name? Because there is no if condition in this step, "if needed" is not accurate.

Reply:
it just means get optional tag name.